### PR TITLE
[Workers] get started update

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-6443

### CODE REVIEW

File: content/workers/get-started/guide.md [hunk 1]
```diff
@@ -47,9 +47,11 @@ $ cd my-project
...
-2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax.
-3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command.
-4. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. Only generated if you indicate `y` to the question `Would you like to use git to manage this Worker? (y/n)` and `y` to the question `No package.json found. Would you like to create one? (y/n)`.
+3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command. Only generated if indicated in `wrangler init` command.
+4. `package-lock.json`: Locks any associated dependencies at the version you installed Wrangler in and that prevents accidental dependency version updating. Only generated if indicated in `wrangler init` command.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
4. [`package-lock.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json): A file that locks any associated dependencies to a specific version and that prevents accidental dependency version updating. Only generated if indicated in `wrangler init` command.
```

File: content/workers/get-started/guide.md [hunk 2]
```diff
@@ -47,9 +47,11 @@ $ cd my-project
...
+5. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+6. `/node_modules`:
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
6. `/node_modules`: Refer to the [NPM documentation](https://docs.npmjs.com/cli/v9/configuring-npm/folders#node-modules) for more information.
```

File: content/workers/get-started/guide.md [hunk 3]
```diff
@@ -47,9 +47,11 @@ $ cd my-project
 In your project directory, `wrangler init` has generated the following files:
 
 1. `wrangler.toml`: Your [Wrangler](/workers/wrangler/configuration/#sample-wranglertoml-configuration) configuration file.
-2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax.
-3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command.
-4. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. Only generated if you indicate `y` to the question `Would you like to use git to manage this Worker? (y/n)` and `y` to the question `No package.json found. Would you like to create one? (y/n)`.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. This is the file where you will write your Worker's code. An `index.ts` file will be generated instead if you indicated `y` to the `Would you like to use TypeScript? (y/n)` question.
```

File: content/workers/get-started/guide.md [hunk 4]
```diff
@@ -47,9 +47,11 @@ $ cd my-project
...
+2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. This is the file where you will write your Worker's code. An `index.ts` file will be generated instead if you indicated `y` to the `Would you like to use TypeScript? (y/n)` question.
+3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command. Only generated if indicated in `wrangler init` command.
+4. [`package-lock.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json): A file that locks any associated dependencies to a specific version and that prevents accidental dependency version updating. Only generated if indicated in `wrangler init` command.
+5. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+6. `/node_modules`: Refer to the [NPM documentation](https://docs.npmjs.com/cli/v9/configuring-npm/folders#node-modules) for more information.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
6. `/node_modules`: Refer to the [NPM documentation on `/node_modules`](https://docs.npmjs.com/cli/v9/configuring-npm/folders#node-modules) for more information.
```

File: content/workers/get-started/guide.md [hunk 5]
```diff
@@ -47,9 +47,11 @@ $ cd my-project
 In your project directory, `wrangler init` has generated the following files:
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
In your project directory, `wrangler init` has generated the following:
```

File: content/workers/get-started/guide.md [hunk 6]
```diff
@@ -44,12 +44,14 @@ To start developing your Worker, `cd` into your new project directory:
...
 
-In your project directory, `wrangler init` has generated the following files:
+In your project directory, `wrangler init` has generated the following:
 
 1. `wrangler.toml`: Your [Wrangler](/workers/wrangler/configuration/#sample-wranglertoml-configuration) configuration file.
-2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax.
-3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command.
-4. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. This is the file where you will write your Worker's code. An `index.ts` file will be generated instead if you indicated `y` to the `Would you like to use TypeScript? (y/n)` question.
+3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command. Only generated if indicated in `wrangler init` command.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command.
```

File: content/workers/get-started/guide.md [hunk 7]
```diff
@@ -44,12 +44,14 @@ To start developing your Worker, `cd` into your new project directory:
...
 ``` 
 
-In your project directory, `wrangler init` has generated the following files:
+In your project directory, `wrangler init` has generated the following:
 
 1. `wrangler.toml`: Your [Wrangler](/workers/wrangler/configuration/#sample-wranglertoml-configuration) configuration file.
-2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax.
-3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command.
-4. `tsconfig.json`: TypeScript configuration that includes [Workers types](https://www.npmjs.com/package/@cloudflare/workers-types). Only generated if indicated in `wrangler init` command.
+2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. This is the file where you will write your Worker's code. An `index.ts` file will be generated instead if you indicated `y` to the `Would you like to use TypeScript? (y/n)` question.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
2. `index.js` (in `/src`): A minimal Hello World Worker written in JavaScript module syntax. This is the file where you will write your code. An `index.ts` file will be generated instead if you indicated `y` to the `Would you like to use TypeScript? (y/n)` question.
```

File: content/workers/get-started/guide.md [hunk 8]
```diff
@@ -44,12 +44,14 @@ To start developing your Worker, `cd` into your new project directory:
...
+3. `package.json`: A minimal Node dependencies configuration file. Only generated if indicated in `wrangler init` command. Only generated if indicated in `wrangler init` command.
+4. [`package-lock.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json): A file that locks any associated dependencies to a specific version and that prevents accidental dependency version updating. Only generated if indicated in `wrangler init` command.
```
Comment (deadlypants1973) [inline content/workers/get-started/guide.md]:
```suggestion
4. [`package-lock.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json): A file that locks any associated dependencies to a specific version and prevents accidental dependency version updating. Only generated if indicated in `wrangler init` command.
```

**Review by pedrosousa — APPROVED** [review]

**Review by rozenmd — APPROVED** [review]
LGTM 👍

**Pull request merged**